### PR TITLE
make the max output line size configurable

### PR DESCRIPTION
**Pull request by thirdeyenick**

### CODE REVIEW

File: modules/shell/command.go [hunk 1]
```diff
@@ -85,12 +86,20 @@ func RunCommandAndGetOutputE(t *testing.T, command Command) (string, error) {
...
-func readStdoutAndStderr(t *testing.T, stdout io.ReadCloser, stderr io.ReadCloser) (string, error) {
+func readStdoutAndStderr(t *testing.T, stdout io.ReadCloser, stderr io.ReadCloser, maxLineSize *int) (string, error) {
 	allOutput := []string{}
 
 	stdoutScanner := bufio.NewScanner(stdout)
 	stderrScanner := bufio.NewScanner(stderr)
 
+	if maxLineSize != nil {
+		stdoutBuf := make([]byte, maxLineSize)
+		stderrBuf := make([]byte, maxLineSize)
```
Comment (tommyknows) [inline modules/shell/command.go]: couldn't we just put this inline to the Buffer creation below, and save a few lines?

```go
if maxLineSize != nil {
    stdoutScanner.Buffer(make([]byte, maxLineSize), maxLineSize)
    stderrScanner.Buffer(make([]byte, maxLineSize), maxLineSize)
}
```

File: modules/terraform/options.go [hunk 2]
```diff
@@ -22,4 +22,5 @@ type Options struct {
 	NoColor                  bool                   // Whether the -no-color flag will be set for any Terraform command or not
 	SshAgent                 *ssh.SshAgent          // Overrides local SSH agent with the given in-process agent
 	NoStderr                 bool                   // Disable stderr redirection
+	OutputMaxLineSize        *int                   // The max size of one line in stdout and stderr (in byte)
```
Comment (tommyknows) [inline modules/terraform/options.go]: `(in bytes)` 🙂

**Review by tommyknows — CHANGES_REQUESTED** [review]

File: modules/shell/command.go (hunk 1 above)
Comment (thirdeyenick) [reply modules/shell/command.go]: done

File: modules/terraform/options.go (hunk 2 above)
Comment (thirdeyenick) [reply modules/terraform/options.go]: fixed

**Review by tommyknows — APPROVED** [review]

**Pull request merged**